src/ngraph_encapsulate_op.cc

```diff
@@ -69,9 +70,27 @@ class NGraphEncapsulateOp : public tf::OpKernel {
     // Create the backend
     if (m_ng_backend == nullptr) {
 #if defined(NGRAPH_EMBEDDED_IN_TENSORFLOW)
-      m_ng_backend = std::make_shared<ng::runtime::interpreter::INTBackend>();
+    	m_ng_backend = ng::runtime::Backend::create("INTERPRETER");
+#elif defined(NGRAPH_TF_BACKEND)
+        auto env_value = std::string(std::getenv("NGRAPH_TF_BACKEND")); 
+	if( env_value.find("INTERPRETER") != string::npos) {
```

Comment:
If the `env_value` is empty, use CPU. Otherwise, directly pass-through the `env_value` to the `ng::runtime::Backend::create`. Since there will be newer backends not listed here.

Reply:
I would also rename `env_value` to something more meaningful like `ng_backend_str`

Reply:
@yxlao .  I'm not so sure about blindly passing in a value that the user prevents.  This will generate an error during backend creation.  If the bridge developers are fine with that, then I'm ok.  I prefer to be more defensive and only allow what we can support.

Reply:
Say the user downloaded a `libnew_backend.so` and put it in the proper location. We want this to work out-of-the-box without compiling the bridge again. Ultimately nGraph backend creator will check whether the backend string is valid or not, here in the bridge, we don't have the information whether a particular backend is present or not. 

So it seems to me that passing the value to `backend::create` and letting the backend creator handle the value would be a better choice. E.g. we could catch an exception here if the `backend::create` throws an exception if there is an error code returned.

Reply:
I would leave this decision to `ngraph-tf` devs